chroma-agent/chroma_agent/action_plugins/agent_setup.py

```diff
@@ -1,7 +1,7 @@
 #
 # INTEL CONFIDENTIAL
 #
-# Copyright 2013-2015 Intel Corporation All Rights Reserved.
+# Copyright 2013-2017 Intel Corporation All Rights Reserved.
```

Comment:
Just an FYI that our copyright is going to change to:

// Copyright (c) 2017 Intel Corporation. All rights reserved.
// Use of this source code is governed by a MIT-style
// license that can be found in the LICENSE file.

So don't worry about updating the year for now.